### PR TITLE
Version v1.1.3 Release

### PR DESCRIPTION
**Pull request by aws-sansalex**
### Fixed
 
- Upgrade workzeug to resolve CVE

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.1.2] - 2024-04-29
```
Comment (mttwise) [inline CHANGELOG.md:8]: set back to 4-29

File: source/modules/cms_vehicle_simulator/source/tests/infrastructure/__snapshots__/test_snapshot/test_cms_vehicle_simulator_snapshot.json [hunk 2]
```diff
@@ -5360,7 +5360,7 @@
         "ssmappuniqueidregistermodule9C5C2C5D"
       ],
       "Properties": {
-        "Create": "{\"action\":\"describeEndpoint\",\"service\":\"Iot\",\"parameters\":{\"endpointType\":\"iot:Data-ATS\"},\"physicalResourceId\":{\"responsePath\":\"endpointAddress\"}}",
+        "Create": "{\"action\":\"describeEndpoint\",\"service\":\"Iot\",\"parameters\":{\"endpointType\":\"iot:Data-ATS\"},\"physicalResourceId\":{\"responsePath\":\"endpointAddress\"},\"logApiResponseData\":true}",
```
Comment (mttwise) [inline source/modules/cms_vehicle_simulator/source/tests/infrastructure/__snapshots__/test_snapshot/test_cms_vehicle_simulator_snapshot.json:5363]: is this correct?

**Review by mttwise — APPROVED** [review]

File: source/modules/cms_vehicle_simulator/source/tests/infrastructure/__snapshots__/test_snapshot/test_cms_vehicle_simulator_snapshot.json (hunk 2 above)
Comment (aws-sansalex) [reply source/modules/cms_vehicle_simulator/source/tests/infrastructure/__snapshots__/test_snapshot/test_cms_vehicle_simulator_snapshot.json:5363]: cdk version upgrade added a field to a construct used in a few places.

**Review by marcburnie — APPROVED** [review]
Approved

**Pull request merged**